src/components/FiltersBox.test.js

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+import {
+  render, cleanup, waitForElementToBeRemoved, fireEvent, wait,
+} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { Simulate } from 'react-dom/test-utils';
+
+import Provider from '../contexts/Provider';
+import Table from './Table';
+import { SW_BASE_API } from '../services/SWapi';
+import mockFetchPromise, { planets } from '../__mocks__/apiMock';
+
+const testTableRender = async () => {
+  jest.spyOn(global, 'fetch').mockImplementation(() => mockFetchPromise);
+
+  const { queryByText, getByRole, queryByTestId } = render(
+    <Provider>
+      <Table />
+    </Provider>,
+  );
+
+  expect(queryByText(/Loading/i)).toBeInTheDocument();
+
```

Comment:
Não precisa de muito espaçamento nos testes. Tente separar por blocos de expect.

```suggestion
const exemplo = getByText(/Loading/i);
expect(exemplo).toBeDefined();

const exemplo2 = getByText(/SWAPI/i);
expect(exemplo2).toBeDefined();
```

Reply:
Eu achava que ficava mais legivel desse jeito, mas nos proximos projetos vou testar da forma que vc falou pra poder comparar, muito obrigado!!